NuGet.config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure why this file changed, did the line ending change maybe?

Reply:
fixed.